sdk/formrecognizer/azure-ai-formrecognizer/tests/test_receipt_async.py

```diff
@@ -400,14 +400,13 @@ async def test_receipt_locale_specified(self, client):
 
     @FormRecognizerPreparer()
     @GlobalClientPreparer()
-    @pytest.mark.skip("the service is returning a different error code")
     async def test_receipt_locale_error(self, client):
         with open(self.receipt_jpg, "rb") as fd:
             receipt = fd.read()
         with pytest.raises(HttpResponseError) as e:
             async with client:
                 await client.begin_analyze_document("prebuilt-receipt", receipt, locale="not a locale")
-        assert "UnsupportedLocale" == e.value.error.code
+        assert "InvalidArgument" == e.value.error.code
```

Comment:
So is this the error code to expect from now on here?

Reply:
Yes, for v2021-09-30-preview this is the error code. Also if the service changes up the error response due to REST feedback during the betas we'll know with this test.